pkg/database/testdata/feature_derivation_case_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testdata
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	old "sqlflow.org/sqlflow/pkg/sql/testdata"
```

Comment:
The "old" go files will be removed later? And will these tests be removed then?

Reply:
Yes, after verifying that the new one works, I plan to remove the tests.